src/Impostor.Plugins.Example/Handlers/GameEventListener.cs

```diff
@@ -24,7 +25,7 @@ public void OnGameStarting(IGameStartingEvent e)
         }
 
         [EventListener]
-        public void OnGameStarted(IGameStartedEvent e)
+        public async void OnGameStarted(IGameStartedEvent e)
```

Comment:
Why is this `async` when its not using any asynchronous tasks?

Reply:
I was using it for testing and forgot to remove it